bench.c

```diff
@@ -59,7 +59,7 @@ static const char *msg_dum = "dummy message";
 static pthread_t pt[MAX_THREAD];
 
 /* block all workers before they are all ready to benchmarking kecho */
-static bool ready;
+static int ready;
```

Comment:
You should explain why you change the type of `ready` within git commit message.

Reply:
If `ready` is used for accumulation, its name would confuse.

Reply:
I have change the commit message. Please check whether it is clear enough, thank you.

Reply:
The name scheme is fairly ambiguous since the word "ready" does not imply the states other than binary form.

Reply:
So maybe changing the variable name to "count" would be better in this scenerio?

Reply:
I have change the name to `ready_cnt`, is it a better name scheme?

Reply:
If you intended to count something, you can use `retry` or `n_retry`.

Reply:
I have changed the name to `n_retry`, thank you!